lib/ts/recipe/passwordless/types.ts

```diff
@@ -170,6 +171,7 @@ export type TypeNormalisedInput = (
       }
     | {
           contactMethod: "EMAIL";
+          emailDelivery: EmailDeliveryTypeInput<TypePasswordlessEmailDeliveryTypeInput>;
```

Comment:
Maybe in normalised config, we can put this outside - since we always want to assign an emailDelivery instance in the recipe class?